### ISSUE TITLE
tar installation instructions are hidden in the build instructions

### ISSUE THREAD

**Issue by mettacrawler** [mi-g/vdhcoapp#40]
I downloaded the Linux tar file with the binaries in it and it did not say to run `sudo /opt/net.downloadhelper.coapp/bin/net.downloadhelper.coapp-linux-64 install`. Later I found that in the instructions for building from the source code. 

There's a link on [the page with the binaries in a tar file](https://www.downloadhelper.net/install-coapp) that says "Extra step required to complete install" but it doesn't actually do anything when I click on that link.

Please don't waste people's time with hidden documentation. Thanks.

**Comment by mi-g** [comment]
Can you check the browser development console to see if an error/warning message is displayed when clicking on the link ?

**Comment by sgbell** [comment]
There is no error thrown in the browser development console when I click the link

**Comment by mi-g** [comment]
Could not reproduce the issue.

**Comment by jimfoye** [comment]
I had this problem today, as I needed to install on Fedora. The link doesn't link to anything.

**Comment by mi-g** [comment]
The *Extra step required to complete install* is not really a link. It triggers a piece of Javascript code that display a text portion that was previously hidden. Could it be that you disabled javascript on some sites including downloadhelper.net ? Also, if you go to https://www.downloadhelper.net/faq , there is a similar mechanism that display/hide FAQ entries. Do you have the same problem there ?

**Comment by jimfoye** [comment]
Ha, I think I know what the problem is. The FAQ portion appears at the bottom of the page. Depending on how much of the page is visible in the browser window, it's not obvious that the additional information has appeared below. You have to scroll to see it. I think the issue here is design - one expects the information to appear immediately below the link, not further down. As is, the only indication that something has changed is that the scroll bar thumb gets a little smaller - if I don't notice that, it feels like nothing has happened when I click the link.

**Comment by mi-g** [comment]
Good catch. That can be the problem for the coapp install page.
However, for the FAQ, the entry should appear right after the clickable faq entry title, and it does so for me: 
![ss-faq](https://user-images.githubusercontent.com/3421255/102922723-ca98fa00-448e-11eb-94d9-5843b7e5e12e.png)
Isn't it what you get when you click on the 3rd faq link ?

**Comment by jimfoye** [comment]
Yes, FAQ page works fine for me, the information appears right below the click target, so there's missing it.
